docs/scaling/kubernetes.txt

```diff
@@ -185,17 +192,46 @@ allows for a better flexibility than passing around config files.
 Storage
 -------
 
+The above example makes use of in-memory storage. This is only suggested to 
+facilitate testing out of CrateDB with Kubernetes. It is not advisable to use
+this form of storage for a production database because all data from this
+directory will be lost if a pod is deleted. More information on this can be
+found `here <https://kubernetes.io/docs/concepts/storage/volumes/>`, under
```

Comment:
you could remove the `under emptyDir` and provide the link as follows https://kubernetes.io/docs/concepts/storage/volumes/#emptydir

Reply:
it doesn't actually take you to that section though...